neutron-org/neutron
# Feat/tx memo

https://p2pvalidator.atlassian.net/browse/LSC-39

See the https://github.com/lidofinance/lido-interchain-staking-contracts/pull/29 for contracts side of memo addition.

Added memo to SubmitTx message and to acknowledgement packet. I've tested it and it returns the memo back.

Test steps:
Use the ./gaia-wasm-zone/demo/readme.md to run test steps, but also need to add logging on acknowledgement to see memo returned back
